google/resource_compute_image_test.go

```diff
@@ -29,6 +29,7 @@ func TestAccComputeImage_basic(t *testing.T) {
 					testAccCheckComputeImageFamily(&image, "family-test"),
 					testAccCheckComputeImageContainsLabel(&image, "my-label", "my-label-value"),
 					testAccCheckComputeImageContainsLabel(&image, "empty-label", ""),
+					testAccCheckComputeImageContainsLicense(&image, "https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx"),
```

Comment:
I think I'd rather see this in its own test, just so we can verify that creating an instance _without_ a license _also_ works.

Reply:
Yup you're right, this was lazy. Updated in d7c1bf9